test-utils/testData/api/parent.kt

```diff
@@ -74,8 +75,15 @@
 // parent of File: a.kt: null
 // parent of Any: Anno
 // parent of Anno: File: a.kt
-// parent of Anno: synthetic constructor for Anno
-// parent of synthetic constructor for Anno: Anno
+// parent of String: String
+// parent of String: param
+// parent of param: <init>
+// parent of Anno: <init>
+// parent of <init>: Anno
+// parent of String: String
+// parent of String: param.getter()
+// parent of param.getter(): param
+// parent of param: <init>
```

Comment:
Does it matter that the test data for the two implementations are different? I'm assuming there are some expected differences between KSP1 and KSP2?

Reply:
It depends but as long as KSP2's output looks right, it is OK. Test data can be forked, for example, in
[KSP2](https://github.com/google/ksp/blob/main/kotlin-analysis-api/src/test/kotlin/com/google/devtools/ksp/test/KSPAATest.kt#L236) and [KSP1](https://github.com/google/ksp/blob/main/compiler-plugin/src/test/kotlin/com/google/devtools/ksp/test/KSPCompilerPluginTest.kt#L206)

Reply:
btw, the tests have to be added to `KSPAATest` and `KSPCompilerPluginTest` like the links in the above to be effective.

Reply:
Ah, you added the cases to existing test cases that are already forked. Please ignore the comments and never mind.